pages/dkp/kommander/2.2/install/configuration/enterprise-catalog/index.md

```diff
@@ -32,6 +32,8 @@ catalog:
 
 Use this configuration when installing or reconfiguring Kommander by passing it to the `dkp install kommander` command:
 
+<p class="message--note"><strong>NOTE: </strong>An alternative to initializing the KUBECONFIG environment variable (by running <code>export KUBECONFIG=cluster_name.conf</code>) is to use the <code>--kubeconfig=cluster_name.conf</code> flag. This ensures that Kommander is installed on the workload cluster.</p>
```

Comment:
```suggestion
<p class="message--note"><strong>NOTE: </strong>To ensures Kommander is installed on the workload cluster, use the <code>--kubeconfig=cluster_name.conf</code> flag as an alternative to KUBECONFIG. </p>
```

Reply:
Agree with Casie that this message is less confusing.  There was one instance missed in the 2.2 doc, I added below.